CTFd/forms/lang.py

```diff
@@ -0,0 +1,10 @@
+from wtforms import RadioField
+
+from CTFd.constants.languages import LANGUAGE_NAMES
+from CTFd.forms import BaseForm
+
+
+class LanguageForm(BaseForm):
+    """Language form used in navbar"""
```

Comment:
This can probably be used elsewhere as well.

Reply:
I changed the comment to say that it is used in core-beta navbar (to help users understand why this form exists).